index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="../Device/css/style.css">
+        <title>Интернет магазин Device</title>
+    </head>
+    <body>
+        <header class="header">
+            <div>
+                <a class="nav_link" href="#">devide</a>
+                <input type="text" name="search">
+                <button type="button">Войти</button>
+                <button type="button">Сравнить</button>
+                <a href="#">Корзина</a>
+            </div>
+            <nav class="nav">                                                
+                <ul class="nav_list">
+                    <li class="#"><a href="#">Каталог товаров</a></li>
+                    <li class="#"><a href="#">Доставка</a></li>
+                    <li class="#"><a href="#">Гарантия</a></li>
+                    <li class="#"><a href="#">Контакты</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <h1>Интернет магазин device</h1>
+            
+            <section class="slider">
+                <h2>Реклама промо товаров</h2>
+                <div class="slider_list">
+                    <div class="slider_item">
+                        <img src="#" alt="Самая длинная палка для селфи">
+                        <p>01</p>
+                        <p class="slider_item">Делай селфи, как Бен Стиллер!</p>
+                        <p class="slider_item">Самая длинная палка для селфи доступна в нашем магазине. Восемь (Восемь Карл!) метров и весом всего 5 килограмм.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Длина палки</dt>
+                            <dd>8,5 м</dd>
+                            <dt>Вес палки</dt>
+                            <dd>5 кг</dd>
+                            <dt>Материал</dt>
+                            <dd>Карбон</dd>
+                        </dl>            
+                    </div>
+                    <div class="slider_item">
+                        <img src="#" alt="фитнес-браслет">
+                        <p>02</p>
+                        <p class="slider_item">Худеем правильно!</p>
+                        <p class="slider_item">Мотивирующие фитнес-браслеты помогут найти в себе силы не пропускать занятия и соблюдать диету.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Без подзарядки</dt>
+                            <dd>48 часов</dd>                            
+                        </dl>            
+                    </div>
+                    <div class="slider_item">
+                        <img src="#" alt="квадрокоптер">
+                        <p>03</p>
```

Comment:
I guess this a pseudo element, think about it after the lecture about styles

Reply:
Made a commnet for myself. Will think about it.